### PR TITLE
[ti_cif3] Ensure event.kind is correctly set for pipeline errors

### PR DESCRIPTION
**Pull request by MakoWish**
## What does this PR do?

Modify ti_cif3 integration to correctly set `event.kind` for pipeline errors and ensure `error.message` is an array.

## Checklist

- [X] I have reviewed tips for building integrations and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's changelog.yml file.
- [ ] I have verified that Kibana version constraints are current according to guidelines.
 
## Author's Checklist

- [ ]  

## Related issues

* For #6582

## Screenshots

### CODE REVIEW

**Comment by MakoWish** [comment]
Forgot to increment the version in manifest. Done.

**Review by efd6 — APPROVED** [review]

File: packages/ti_cif3/manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-format_version: 1.0.0
+format_version: 0.8.0
```
Comment (efd6) [inline packages/ti_cif3/manifest.yml:1]: This is not right.

```suggestion
format_version: 1.0.0
```

Comment (MakoWish) [reply packages/ti_cif3/manifest.yml:1]: It was listed as beta, so didn't think to bump the major version up.

**Review by efd6 — APPROVED** [review]

**Pull request merged**